extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -118,6 +118,13 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 		}
 
 		existingSettings.unshift({ label: constants.createNewLocalAppSettingWithIcon, isCreateNew: true });
+		let index = existingSettings.findIndex(s => s.label === constants.sqlConnectionStringSetting);
+		if (index > -1) {
+			let sqlConnectionString = existingSettings.splice(index, 1)[0];
+			existingSettings.unshift(sqlConnectionString);
+		} else {
+			existingSettings.unshift({ label: constants.sqlConnectionStringSettingWithIcon, isCreateNew: true });
```

Comment:
I'm trying to decide if this is enough to have the + or if we want to do something else. It just seems kind of hard to parse for someone who doesn't know what that value is. 

Can you try adding (new) as the secondary text and see how that looks?

@dzsquared @chlafreniere Thoughts? Maybe I'm just overthinking it but right now I'm just worried this is going to be confusing for users.

Reply:
I don't think you're overthinking this. Let's try secondary text.

In addition, I think that the scenario is then clearer, because in the first screenshot, we're listing:
- <Existing setting>
- <New setting>
- <Existing setting>
- <Existing setting>

Which is a little hard to follow. We're trying to help customers choose the logical one (if it exists), but if I were new to this, I'd be confused about the grouping.

Net-net I think that secondary text can be super helpful here.

Reply:
I agree that it can be confusing for users. This is what it looks like with secondary text:
<img width="631" alt="Screen Shot 2021-11-12 at 4 23 43 PM" src="https://user-images.githubusercontent.com/23462877/141598026-ed818a6e-0184-4f5d-b5d4-5bd0cf987569.png">
Should we go with this?

Reply:
Maybe `(new)` instead to make it even more clear it's not a part of the name

Reply:
<img width="628" alt="Screen Shot 2021-11-12 at 4 40 03 PM" src="https://user-images.githubusercontent.com/23462877/141598716-78840692-8277-4ac2-b799-f137e58f15b3.png">

Reply:
(New) with parenthesis reads more easily to me

Reply:
I think this currently sends mixed signals.  "AzureWebJobsStorage" and "FUNCTIONS_WORKER_RUNTIME" are obviously existing settings, and "Create new local app setting" is clear enough that it's creating a new one, but "+ SqlConnectionString (New)" is an odd blend.  Maybe it should be "Create new 'SqlConnectionString' setting" or be a second selection or autofilled name after the user selects "+ Create new local app setting".

Reply:
I like the idea of having "SqlConnectionString" be autofilled after "+ Create new local app setting".

Reply:
Yeah that seems reasonable. It's an extra click I was hoping to avoid but I agree it's a little messy as is